smart-contracts/contracts-common/concordium-contracts-common/src/types.rs

```diff
@@ -1048,6 +1048,21 @@ impl AccountAddress {
             None
         }
     }
+
+    /// Get the `n-th` alias of an address. There are 2^24 possible aliases.
```

Comment:
Please also make `get_alias` const and use `get_alias_unchecked` inside it to avoid repetition.